packages/server/src/models/user.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose, { Document, Model } from 'mongoose';
```

Comment:
use the below syntax:
`import mongoose, { type Document, Model } from 'mongoose';`